src/Groups/directproducts.jl

```diff
@@ -162,22 +167,25 @@ It is not defined for proper subgroups of direct products.
 # Examples
 ```jldoctest
 julia> H = symmetric_group(3)
-Sym(3)
+Symmetric group of degree 3 and order 6 with 2 generators
+  (1,2,3)
+  (1,2)
 
 julia> K = symmetric_group(2)
-Sym(2)
+Symmetric group of degree 2 and order 2 with 1 generator
+  (1,2)
 
 julia> G = direct_product(H, K)
-Direct product of
- Sym(3)
- Sym(2)
+Symmetric group of degree 3 and order 6 x Symmetric group of degree 2 and order 2
+with 3 generators
+  (1,2,3)
+  (1,2)
+  (4,5)
 
 julia> inj1 = canonical_injection(G, 1)
 Group homomorphism
-  from Sym(3)
-  to direct product of
-   Sym(3)
-   Sym(2)
+  from symmetric group of degree 3 and order 6
+  to symmetric group of degree 3 and order 6 x Symmetric group of degree 2 and order 2
```

Comment:
Also not happy about this. I think now better to keep it as it was, or change to one of these
```suggestion
  to Sym(3) x Sym(2)
```
or
```suggestion
  to direct product of
    symmetric group of degree 3 and order 6
    symmetric group of degree 2 and order 2
```